TAScheduler/Managment/UserManagement.py

```diff
@@ -24,32 +56,59 @@ def createUser(self):
     # UserId(in) - Id of the user
     # User Name(in) - Name of the user
     # User Contact(in) - Contact of the user
-    # User SSN(in) - SSN of the user
     # User Address(in) - Address of the user
     # User Password(in) - Password of the user
+    # User Email(in) - Email of the user
     # User Type(in) - Type of the user
-    def editUser(self):
-        pass
+    def editUser(self, Id, name, contact, email, address, password, usertype):
+        if(not(isinstance(Id, int))):
+            raise TypeError("Id entered is not of type int")
+        if(not(isinstance(name, str))):
+            raise TypeError("Name entered is not of type str")
+        if(not(isinstance(contact, str))):
+            raise TypeError("Contact entered is not of type str")
+        if(not(isinstance(email, str))):
+            raise TypeError("Email entered is not of type str")
+        if(not(isinstance(address, str))):
+            raise TypeError("Address entered is not of type str")
+        if(not(isinstance(password, str))):
+            raise TypeError("Address entered is not of type str")
+        if (not(usertype in ["SUPERVISOR", "INSTRUCTOR", "TA"])):
+            raise TypeError("SSN entered is not of type int")
```

Comment:
As I mentioned before, SSN is from an obsolete version of the design. The error this should raise should be about userType, not ssn.